app/views/devise/registrations/new.html.erb

```diff
@@ -34,7 +34,7 @@
         </div>
 
         <div class="form-group">
-          <%= f.label :password_confirmation, :class => "col-sm-2 control-label" %>
+          <%= f.label :password_confirmation, "Passwort bestätigung", :class => "col-sm-2 control-label" %>
```

Comment:
When it's all one word, it's too big to fit in the label space, do you want me to change the label column to col-sm-3 (and make the input fields smaller) ?